gauntlet/gauntlet-terra-cw-plus/package.json

```
@@ -14,16 +14,16 @@
   ],
   "scripts": {
     "preinstall": "node ../../scripts/require-yarn.js",
-    "gauntlet": "ts-node ./src/index.ts",
+    "gauntlet": "ts-node src/index.ts",
     "lint": "tsc",
     "test": "SKIP_PROMPTS=true jest --runInBand",
     "test:coverage": "yarn test --collectCoverage",
     "test:ci": "yarn test --ci",
-    "lint:format": "yarn prettier --check ./src",
-    "format": "yarn prettier --write ./src",
+    "lint:format": "yarn prettier --check src",
+    "format": "yarn prettier --write src",
     "clean": "rm -rf ./dist/ ./bin/",
     "build": "yarn clean && tsc",
-    "bundle": "yarn build && pkg ."
+    "bundle": "yarn build && pkg "
```

Not sure if the trailing dot is required, but now there is a trailing space here:

```suggestion
    "bundle": "yarn build && pkg"
```